clang/test/CheckedCRewriter/refarrsubscript.c

```diff
@@ -16,7 +16,9 @@ int **bar(struct foo *p) {
 //CHECK_NOALL: struct foo { int **b; int n; };
 //CHECK_NOALL-NEXT: int ** bar(_Ptr<struct foo> p) {
 //CHECK_NOALL-NEXT:  _Ptr<int> n =  &p->n;
-//CHECK_ALL: struct foo { _Array_ptr<_Ptr<int>> b; int n; };
+//CHECK_ALL: struct foo { _Array_ptr<_Ptr<int>> b;
+//CHECK_ALL-NEXT: int n;
+//CHECK_ALL-NEXT: ; ; };
```

Comment:
This seems non-ideal. Why is it now breaking across multiple lines whereas previously it was staying on the same line?